packages/create-platformatic/src/service/create-service-cli.mjs

```diff
@@ -51,12 +51,28 @@ const createPlatformaticService = async (_args) => {
 
   const projectDir = await askProjectDir(logger, '.')
 
+  const { runPackageManagerInstall, useTypescript } = await inquirer.prompt([{
+    type: 'list',
+    name: 'runPackageManagerInstall',
+    message: `Do you want to run ${pkgManager} install?`,
+    default: true,
+    choices: [{ name: 'yes', value: true }, { name: 'no', value: false }]
+  }, {
+    type: 'list',
+    when: !args.typescript,
+    name: 'useTypescript',
+    message: 'Do you want to use TypeScript?',
+    default: args.typescript,
+    choices: [{ name: 'yes', value: true }, { name: 'no', value: false }]
+  }])
```

Comment:
idea: We have the duplicated prompt fields in `db` and `service` CLI. Is this better to extract these in another file and manage it in a single place?

Reply:
I think so, go ahead!

Reply:
Thanks! I extracted these two cli options first as part of this PR to not increase the scope of changes and wrote unit tests accordingly. If this approach is fine, I can create a separate PR to collect all of the options in a single place and write unit tests for validating contents. Please have a look when you are available 🙇
[53e7b0b](https://github.com/platformatic/platformatic/pull/757/commits/53e7b0b42c09e1c30e3b7201cb448bd41f62d675)